client/my-sites/theme/main.jsx

```diff
@@ -87,6 +87,8 @@ class ThemeSheet extends Component {
 			PropTypes.string,
 			PropTypes.bool, // happens if no content: false
 		] ),
+		seoTitle: PropTypes.string,
+		seoDescription: PropTypes.string,
```

Comment:
These two `propTypes` are incorrect and unnecessary. There're no `seoTitle` and `seoDescription` being past to the component directly.
What happens is they get `borrowed` and spread from the `theme` object [on this line](https://github.com/Automattic/wp-calypso/blob/update/update-theme-metadata/client/my-sites/theme/main.jsx#L887). This, likely, contains the raw data from the API which don't seem to be listed here (it'd be a long list).